spec/requests/api/v1/digital_objects_spec.rb

```diff
@@ -1,9 +1,135 @@
 require 'rails_helper'
 
-RSpec.describe "Digital Objects API endpoint", type: :controller do
+RSpec.describe "Digital Objects API endpoint", type: :request do
+  let(:authorized_object) { FactoryBot.create(:digital_object_test_subclass, :with_sample_data, :with_lincoln_project) }
+  let(:authorized_project) { authorized_object.projects.to_a[0] }
+
   describe 'GET /api/v1/digital_objects/:id' do
-    it "return a single Digital Object with the expected fields" do
-      skip
+    include_examples 'requires user to have correct permissions' do
+      let(:request) do
+        get "/api/v1/digital_objects/#{authorized_object.uid}"
+      end
+    end
+
+    context "logged in" do
+      before do
+        sign_in_project_contributor to: :read_objects, project: authorized_project
+        get "/api/v1/digital_objects/#{authorized_object.uid}"
+      end
+
+      it 'returns 200' do
+        expect(response.status).to be 200
+      end
+      it "return a single Digital Object with the expected fields" do
+        expect(JSON.parse(response.body)).to include('uid' => authorized_object.uid)
```

Comment:
I've been using `json_spec` to do the json matching in case that's easier.

https://github.com/collectiveidea/json_spec

Reply:
Will read up on it!